core/src/main/java/nl/nn/adapterframework/senders/XsltSender.java

```diff
@@ -456,7 +456,7 @@ public void setSkipEmptyTags(boolean b) {
 	}
 
 	/**
-	 * If set to <code>1</code> xslt processor 1.0 (org.apache.xalan) will be used, otherwise xslt processor 2.0 (net.sf.saxon). <code>0</code> will auto detect
+	 * If set to <code>1</code> xslt processor 1.0 (org.apache.xalan) will be used, otherwise xslt processor 2.0 (net.sf.saxon). <code>0</code> will auto detect. This class supports up to and including XSLT version 3.0
```

Comment:
Actually this comment should be updated to say that the Saxon processor will be used when setting this parameter to 2 or 3.